### DISCUSSION TITLE
Nextcloud in maintenance mode - NC 26.0.1 recognize v4.0.1

### DISCUSSION THREAD

**Discussion by Ymetro** [nextcloud/recognize#870] [Q&A]
### Which version of recognize are you using?

4.0.1

### Enabled Modes

Object recognition, Face recognition, Video recognition, Music recognition

### TensorFlow mode

WASM mode

### Which Nextcloud version do you have installed?

Nextcloud Hub 4 (26.0.1)

### Which Operating system do you have installed?

Slackware 15.0 - Unraid 6.11.5 - kernel 5.19.17-Unraid

### Which Docker container are you using to run Nextcloud? (if applicable)

Nextcloud Official v26.0.1

### How much RAM does your server have?

32GiB

### What processor Architecture does your CPU have?

Intel i7 4790K X86_64

### Describe the Bug

Never have ran Recognize so can not answer mandatory questions about Enabled Modes and Tensor Flow mode, so please ignore those answers. 
App gives internal server error in Administrator Settings page in Nextcloud after upgrade of Nextcloud. 
Update of Nextcloud Docker image from 26.0.0.1 to 26.0.1 (if memory recalls correctly) gave Nextcloud the maintenance mode enabled status. 
Manual upgrade of Nextcloud gave error:
"Updating <recognize> ...
Repair error: RecursiveDirectoryIterator::__construct(/var/www/html/custom_apps/recognize/node_modules/ffmpeg-static/): Failed to open directory: No such file or directory
Updated <recognize> to 4.0.1"

### Expected Behavior

Normal run 

### To Reproduce

Install Recognize in official Nextcloud Docker, upgrade Nextcloud Docker image to Nextcloud 26.0.1 and update Recognize for Nextcloud app to 4.0.1. Nextcloud then went in maintenance mode. After manual upgrade and disable of maintenance mode, Nextcloud was running again, but Recognize did not. 

### Debug log

I found some hints in the tail of the Docker rotational log. Added as attachment file. 
[nextcloudtail.log](https://github.com/nextcloud/recognize/files/11414817/nextcloudtail.log)

**Comment by marcelklehr** [comment]
> 26.0.1

Can you try updating to Nextcloud v26.0.2 and recognize v4.1.0 ?
